src/components/Menu/Menu.module.scss

```diff
@@ -1,14 +1,117 @@
-@media (min-width: 768px) {
-}
-@media (min-width: 480px) and (max-width: 768px) {
+.Menu {
+    width: 280px;
+
+    position: fixed;
+    top: 0;
+    right: 0;
+
+    overflow-y: visible;
+
+    .MenuOpen {
+        color: white;
+        backdrop-filter: blur(25px);
+
+        border-bottom-left-radius: 50px;
+        text-align: center;
+        font-size: 32px;
+        font-family: Roboto, sans-serif;
+
+        .MenuHeader {
+            text-decoration: underline;
+        }
+
+        .MenuItem {
+            margin: 32px 0;
+            &:hover {
+                color: red;
+                cursor: pointer;
+            }
+        }
+
+        .BokaBordBtn {
```

Comment:
I think language consistency should be enforced, now it's mixed with Swedish & English.